src/components/EditGrower.js

```diff
@@ -161,12 +161,22 @@ const EditGrower = (props) => {
               ))}
             </Grid>
           ))}
+          <TextField
+            id="outlined-multiline-static"
+            className={classes.textInput}
+            label="About Me"
+            multiline
+            rows={3}
+            placeholder="About Me"
+            value={getValue('about')}
+          />
```

Comment:
There's nothing here to handle changes to `about` and persist the new value.
You will get this for free using the `inputs` array.